gobblin-temporal/src/main/java/org/apache/gobblin/temporal/cluster/GobblinTemporalClusterManager.java

```diff
@@ -238,16 +230,7 @@ public synchronized void stop() {
 
     LOGGER.info("Stopping the Gobblin Cluster Manager");
 
-    if (this.idleProcessThread != null) {
```

Comment:
Temporal needs to run in yarn mode. And so we do not need the standalone cluster / bare metal code